tasks/docker_docs/Dockerfile

```diff
@@ -15,4 +15,4 @@ USER kibana
 RUN git clone --depth 1 https://github.com/elastic/docs.git /home/kibana/docs_builder
 
 WORKDIR /home/kibana/docs_builder
-CMD git pull origin master && ./build_docs.pl --doc /home/kibana/ascii_docs/index.asciidoc --out /home/kibana/html_docs --chunk=1
+CMD git pull origin main && ./build_docs.pl --doc /home/kibana/ascii_docs/index.asciidoc --out /home/kibana/html_docs --chunk=1
```

Comment:
This file has a lot of legacy references. Would you be able to create a follow-up issue for us to clean this up/delete?

Reply:
Created: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1784